pkg/cloud/gcpprovider.go

```diff
@@ -366,7 +366,7 @@ func (gcp *GCP) ExternalAllocations(start string, end string, aggregators []stri
 				service.description as service,
 				TO_JSON_STRING(labels) as keys,
 				SUM(cost) as cost
-			FROM  %s
+			FROM  "%s"
```

Comment:
Based on my tests this needs to be ` back ticks not "

Reply:
That is true. https://stackoverflow.com/questions/51143932/syntax-error-unexpected-string-literal-93868086-ga-sessions-at-1244-big

Thank you for the review!